apps/student/src/app/documents/[documentId]/page.tsx

```
@@ -0,0 +1,605 @@
+                      ref={fileInputRef}
+                      type="file"
+                      accept=".pdf,.hwp,application/pdf"
+                      className="hidden"
+                      onChange={handleFileChange}
+                    />
+                    <div
+                      onClick={handleUploadClick}
+                      onDragOver={handleDragOver}
+                      onDrop={handleDrop}
+                      className="flex h-32 w-full cursor-pointer flex-col items-center justify-center rounded-lg border-2 border-gray-200 border-dashed bg-gray-50 text-gray-400 transition-colors hover:bg-gray-100"
+                    >
```

Biome (2.4.15)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

(lint/a11y/useKeyWithClickEvents)